sorc/build.rdas

```diff
@@ -51,13 +51,19 @@ cp ../_workaround_/RDASApp/ObsErrorFactorPressureCheck.h sorc/ufo/src/ufo/filter
 # https://github.com/JCSDA-internal/ufo/pull/3622 by Sijie
 cp ../_workaround_/RDASApp/sfccorrected/EvalSurfaceTemperature.cc sorc/ufo/src/ufo/operators/sfccorrected/EvalSurfaceTemperature.cc
 cp ../_workaround_/RDASApp/sfccorrected/EvalSurfaceTemperature.h sorc/ufo/src/ufo/operators/sfccorrected/EvalSurfaceTemperature.h
+cp ../_workaround_/RDASApp/sfccorrected/EvalSurfaceHumidity.cc sorc/ufo/src/ufo/operators/sfccorrected/EvalSurfaceHumidity.cc
+cp ../_workaround_/RDASApp/sfccorrected/EvalSurfaceHumidity.h sorc/ufo/src/ufo/operators/sfccorrected/EvalSurfaceHumidity.h
+cp ../_workaround_/RDASApp/sfccorrected/EvalSurfaceWind.cc sorc/ufo/src/ufo/operators/sfccorrected/EvalSurfaceWind.cc
+cp ../_workaround_/RDASApp/sfccorrected/EvalSurfaceWind.h sorc/ufo/src/ufo/operators/sfccorrected/EvalSurfaceWind.h
+cp ../_workaround_/RDASApp/sfccorrected/ObsSfcCorrected.cc sorc/ufo/src/ufo/operators/sfccorrected/ObsSfcCorrected.cc
 cp ../_workaround_/RDASApp/sfccorrected/ObsSfcCorrectedParameters.h sorc/ufo/src/ufo/operators/sfccorrected/ObsSfcCorrectedParameters.h
+cp ../_workaround_/RDASApp/sfccorrected/CMakeLists.txt sorc/ufo/src/ufo/operators/sfccorrected/CMakeLists.txt
 
 # workaround for direct radar refl DA
 # https://github.com/JCSDA-internal/mpas-jedi/pull/1045 by Yongming Wang
-cp ../_workaround_/RDASApp/direct_refl/mpas_fields_mod.F90 sorc/mpas-jedi/src/mpasjedi/Fields/mpas_fields_mod.F90
-cp ../_workaround_/RDASApp/direct_refl/mpas_state_mod.F90 sorc/mpas-jedi/src/mpasjedi/State/mpas_state_mod.F90
-cp ../_workaround_/RDASApp/direct_refl/mpas2ufo_vars_mod.F90 sorc/mpas-jedi/src/mpasjedi/Utilities/mpas2ufo_vars_mod.F90
+cp ../_workaround_/RDASApp/mpas-jedi/mpas_fields_mod.F90 sorc/mpas-jedi/src/mpasjedi/Fields/mpas_fields_mod.F90
+cp ../_workaround_/RDASApp/mpas-jedi/mpas_state_mod.F90 sorc/mpas-jedi/src/mpasjedi/State/mpas_state_mod.F90
+cp ../_workaround_/RDASApp/mpas-jedi/mpas2ufo_vars_mod.F90 sorc/mpas-jedi/src/mpasjedi/Utilities/mpas2ufo_vars_mod.F90
```

Comment:
I assume your changes to mpas-jedi does not affect the direct_refl capability. But it will be good if you can confirm this. Thanks!

Reply:
Thanks for the confirmation!